src/main/java/ch/njol/skript/expressions/ExprElement.java

```diff
@@ -175,6 +192,38 @@ protected T[] get(Event event) {
 		return elementArray;
 	}
 
+	@SuppressWarnings("unchecked")
+	private T @Nullable [] getFromQueue(Event event) {
+		SkriptQueue queue = (SkriptQueue) expr.getSingle(event);
+		if (queue == null)
+			return null;
+		int startIndex = 0, endIndex = 0;
+		if (this.startIndex != null) {
+			startIndex = this.startIndex.getOptionalSingle(event).orElse(1);
```

Comment:
Hmm do you think we should instead abort if the startIndex/endIndex expressions return null?
For something like `set {_list::*} to the first {_unset_variable} elements of {_queue}`, I'm not sure that we want to fallback to popping the first element.